accepted/rfc0000-minimum-rustc.md

```diff
@@ -0,0 +1,91 @@
+# Summary
+[summary]: #summary
+
+Set a formal policy for the minimum supported Rust version in `rayon` and
+`rayon-core`.
+
+- We will support *at least* one year of Rust compilers, meaning that our
+  chosen minimum will be at least one year old at the time we release any
+  rayon crate requiring that.
+
+- Increases to our minimum supported Rust will also require a minor increase in
```

Comment:
The phrasing here is confusing to me -- ah, I guess the point is that it requires a minor increase, and cannot be done in a patch release?

Reply:
Yes - I can add "not a patch release" in there.